trpcultivate_phenoshare/trpcultivate_phenoshare.module

```diff
@@ -27,3 +27,24 @@ function trpcultivate_phenoshare_help($route_name, RouteMatchInterface $route_ma
     default:
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function trpcultivate_phenoshare_form_alter(&$form, &$form_state, $form_id) {
+  // This hook will omit from Tripal Importer Plugin.
+  // - Field file upload.
+  // - Submit button.
+  // - Advanced Option (Database options).
+  if(isset($form['importer_plugin_id']['#value']) 
+    && $form['importer_plugin_id']['#value'] == 'trpcultivate-phenotypes-share') {
+      // Disable selection of previously uploaded file. 
+      $form['file']['file_upload_existing'] = [];
+      // Form will implement a form button that can be re-named
+      // relevant to each stage.
+      $form['button'] = [];
+      // Exclude the database option (advanced options) since phenotypes
+      // schema is always installed in chado database (default schema).
+      $form['advanced']['#access'] = FALSE;
```

Comment:
It is never right to exclude the option for the user to choose the Chado schema. Please do not ever disable this. Rather you will want to work it into your Stage 1. You can theme it to blend in more if you want but it should not be removed.

Reply:
My goal is to keep the importer form as simple as possible, no advanced configurations or other unnecessary field elements. If this module is closely tied to chado I think, there shouldn't be the need for this field... but then maybe I am missing a detail if we could discuss more about this. The current PR includes this advanced option

Reply:
This field doesn't allow them to select something other than chado. Rather it lets them select which install of chado they want to use. 

For example, a site may decide to have a chado install for private data (named `private_biodata`) and a separate chado install for public data (named `public_biodata`). In this example, a curator would choose from this drop down either `private_biodata` or `public_biodata` based on whether the researchers ability to release the data. Both `private_biodata` and `public_biodata` would be Chado schema with all the same tables.

Another example might be having one chado install for rough or in-progress data and another for published data only. This way they could use another module or postgresql itself to ensure the published data is read-only once inserted.

While this module doesn't need to support any of this advanced functionality, it needs to allow the Importer API to add things like this to the form so that others can implement it. You just need to make sure you do not unset anything from the form created by the importer API. Instead you can keep the simplicity of this form by themeing the advanced fieldset to be collapsed by default and less conspicuous.